modules/rest-api/pages/rest-cluster-autofailover-enable.adoc

```diff
@@ -77,6 +79,10 @@ This parameter is optional, and is only available in Couchbase Enterprise Editio
 The value can be either `true` (the default) or `false`.
 The parameter and its value are ignored, if `enabled` is set to `false`.
 
+* `failoverPreserveDurabilityMajority`.
+Can be `true` (the default) or `false`.
+If this setting is `true`, and the node to be automatically failed over is a Data-Service node that hosts data for a bucket whose Minimum Durability Level is greater than `none`; auto-failover occurs only if xref:learn:data/durability.adoc#majority[Majority] can be maintained across the nodes on which the bucket's data resides.
```

Comment:
Same comment about majority from above

Reply:
> * `failoverPreserveDurabilityMajority`.
> Can be `true` (the default) or `false`.

=> No. In 7.2.0, it is **false** by default:

[Preserve durable writes is unchecked by default](https://issues.couchbase.com/secure/attachment/211446/Screenshot%202023-06-27%20at%2014.55.43.png)

Please make sure the upcoming doc page does not contradict what is inside the current UI.

Thanks.